src/dotnet/Chat.Service/Places.cs

```diff
@@ -153,30 +196,8 @@ public virtual async Task OnPromoteToOwner(Places_PromoteToOwner command, Cancel
 
     public virtual async Task OnDelete(Places_Delete command, CancellationToken cancellationToken)
     {
-        var (session, placeId) = command;
-        var place = await Get(session, placeId, cancellationToken).ConfigureAwait(false);
-        if (place == null)
-            return;
-
-        place.Rules.Require(PlacePermissions.Owner);
-        var contacts = await Contacts.ListIds(session, placeId, cancellationToken).ConfigureAwait(false);
-        foreach (var contact in contacts) {
-            var chatId = contact.ChatId;
-            if (chatId.IsPeerChat(out _))
-                continue; // Ignore peer chats
-            var chat = await Chats.Get(session, chatId, cancellationToken).ConfigureAwait(false);
-            if (chat != null && OrdinalEquals(Constants.Chat.SystemTags.Welcome, chat.SystemTag)) {
-                var resetChatTagCommand = new Chats_Change(session, chatId, null, new Change<ChatDiff> {
-                    Update = new ChatDiff { SystemTag = Symbol.Empty }
-                });
-                await Commander.Call(resetChatTagCommand, true, cancellationToken).ConfigureAwait(false);
-            }
-            var deleteChatCommand = new Chats_Change(session, chatId, null, new Change<ChatDiff> { Remove = true });
-            await Commander.Call(deleteChatCommand, true, cancellationToken).ConfigureAwait(false);
-        }
-
-        var deleteRootChatCommand = new Chats_Change(session, place.Id.ToRootChatId(), null, new Change<ChatDiff> { Remove = true });
-        await Commander.Call(deleteRootChatCommand, true, cancellationToken).ConfigureAwait(false);
+        var placeChangeCommand = new Places_Change(command.Session, command.PlaceId, null, Change.Remove<PlaceDiff>());
```

Comment:
if Places_Delete command for backward compatibility only, then maybe we need to mark it properly

Reply:
Done